src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4185,7 +4185,10 @@ .FORWARDHELPCATEGORY Cmdlet
             $customPagerCommandLine = $env:PAGER
 
             # Split the command line into tokens, respecting quoting.
-            $customPagerCommand, $customPagerCommandArgs = & { Write-Output -- $customPagerCommandLine }
+            $errs = $null
+            $tokens = [System.Management.Automation.PSParser]::Tokenize($customPagerCommandLine, [ref]$errs)
```

Comment:
My only question is whether is makes sense to handle all the tokens, rather than just the first two. I don't know much about this functionality though, so perhaps not

Reply:
Hey @rjmholt,
I briefly looked into it and I'm guessing we'd rather use a powershell function specifically designed to split the command  line into it's command and arguments. Having said that I haven't found a powershell function that does that and is a part of PSCore.
Thoughts ? :)
I am of the opinion that using Tokenize and only keeping the first two tokens is the best option.